designsafe/apps/rapid/opentopography_data/scripts/README.md

```diff
@@ -0,0 +1,41 @@
+# OpenTopo Data Centerpoint Generator
```

Comment:
This could be dropped now as they have improved their api so these scripts are not used anymore. They were used during summer of 2024 as we expored using OpenTopo